### PR TITLE
Prepared release 1.1.0

### PR DESCRIPTION
**Pull request by oskarkulinski**
Changed version to 1.1.0 and added "Egzamin" as viable grade class type

### CODE REVIEW

File: README.md
Comment (filiera) [inline README.md:1]: nice

File: lib/pages/grades.dart
Comment (filiera) [inline lib/pages/grades.dart:1]: nareszcie

File: pubspec.lock
Comment (filiera) [inline pubspec.lock:1]: nwm co to :(

**Review by filiera — APPROVED** [review]
LGTM

File: lib/pages/grades.dart
Comment (oskarkulinski) [reply lib/pages/grades.dart:1]: Tak

File: pubspec.lock
Comment (oskarkulinski) [reply pubspec.lock:1]: Nikt nie wie

**Pull request merged**